app/models/review.rb

```diff
@@ -5,9 +5,13 @@ class Review < ActiveRecord::Base
   has_many :downvotes, dependent: :destroy
   has_many :upvotes, dependent: :destroy
 
-  validates :user, presence: true
-  validates :body, presence: true
-  validates :game_rating, numericality: {
+  validates :user, presence: true,
+    :uniqueness => { :scope => :game,
+    :message => "may only write one review per game." }
```

Comment:
Align the elements of a hash literal if they span more than one line.<br>Use the new Ruby 1.9 hash syntax.